contracts/actions/curve/gauge/CurveMintCrvMany.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "../../ActionBase.sol";
+import "../helpers/CurveHelper.sol";
+import "../../../utils/TokenUtils.sol";
+import "../../../utils/SafeMath.sol";
+
+contract CurveMintCrvMany is ActionBase, CurveHelper {
```

Comment:
Can we consolidate  CurveMintCrv and CurveMintCrvMany. The inputs can be address[8] and if only 1 addresses is filed it will mint only 1 and if more than 1 address is filled it will mint many.